<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CDialogIrcSettings</class>
 <widget class="QDialog" name="DialogIrcSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>392</width>
    <height>281</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Irc Settings</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../Resource.qrc">
    <normaloff>:/Resource/Generic/Settings.png</normaloff>:/Resource/Generic/Settings.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="margin">
    <number>2</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidgetSettings">
     <property name="currentIndex">
      <number>3</number>
     </property>
     <widget class="QWidget" name="tabGeneral">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
       </property>
       <property name="horizontalSpacing">
        <number>4</number>
       </property>
       <property name="verticalSpacing">
        <number>4</number>
       </property>
       <property name="margin">
        <number>4</number>
       </property>
       <item row="1" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>&amp;UI:</string>
         </property>
         <property name="buddy">
          <cstring>comboBoxLayout</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="comboBoxLayout">
         <item>
          <property name="text">
           <string>Tabs</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Tree</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>&amp;Scrollback:</string>
         </property>
         <property name="buddy">
          <cstring>spinBoxBlockCount</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QSpinBox" name="spinBoxBlockCount">
         <property name="maximum">
          <number>9999</number>
         </property>
         <property name="singleStep">
          <number>10</number>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>&amp;Messages:</string>
         </property>
         <property name="buddy">
          <cstring>checkBoxTimeStamp</cstring>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QCheckBox" name="checkBoxTimeStamp">
           <property name="text">
            <string>Timestamp</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBoxStripNicks">
           <property name="text">
            <string>Strip nicks</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="0" colspan="2">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QCheckBox" name="checkBoxConnectOnStartup">
           <property name="text">
            <string>Connect on Startup</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="4" column="0" colspan="2">
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Timestamp Format:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditTimestampFormat"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabShortcuts">
      <attribute name="title">
       <string>Shortcuts</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="spacing">
        <number>4</number>
       </property>
       <property name="margin">
        <number>4</number>
       </property>
       <item>
        <widget class="CWizardTreeWidget" name="treeWidgetShortcuts">
         <property name="inputMethodHints">
          <set>Qt::ImhNoAutoUppercase</set>
         </property>
         <property name="rootIsDecorated">
          <bool>false</bool>
         </property>
         <property name="uniformRowHeights">
          <bool>true</bool>
         </property>
         <property name="itemsExpandable">
          <bool>false</bool>
         </property>
         <property name="allColumnsShowFocus">
          <bool>true</bool>
         </property>
         <column>
          <property name="text">
           <string>Description</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Shortcut</string>
          </property>
         </column>
         <item>
          <property name="text">
           <string>Navigate</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="flags">
           <set>ItemIsSelectable|ItemIsEnabled</set>
          </property>
          <item>
           <property name="text">
            <string>Up</string>
           </property>
           <property name="flags">
            <set>ItemIsSelectable|ItemIsEditable|ItemIsEnabled</set>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Down</string>
           </property>
           <property name="flags">
            <set>ItemIsSelectable|ItemIsEditable|ItemIsEnabled</set>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Left</string>
           </property>
           <property name="flags">
            <set>ItemIsSelectable|ItemIsEditable|ItemIsEnabled</set>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Right</string>
           </property>
           <property name="flags">
            <set>ItemIsSelectable|ItemIsEditable|ItemIsEnabled</set>
           </property>
          </item>
         </item>
         <item>
          <property name="text">
           <string>Next unread</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="flags">
           <set>ItemIsSelectable|ItemIsEnabled</set>
          </property>
          <item>
           <property name="text">
            <string>Up</string>
           </property>
           <property name="flags">
            <set>ItemIsSelectable|ItemIsEditable|ItemIsEnabled</set>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Down</string>
           </property>
           <property name="flags">
            <set>ItemIsSelectable|ItemIsEditable|ItemIsEnabled</set>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Left</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="flags">
            <set>ItemIsSelectable|ItemIsEditable|ItemIsEnabled</set>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Right</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="flags">
            <set>ItemIsSelectable|ItemIsEditable|ItemIsEnabled</set>
           </property>
          </item>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabColors">
      <attribute name="title">
       <string>Colors</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <property name="spacing">
        <number>4</number>
       </property>
       <property name="margin">
        <number>4</number>
       </property>
       <item>
        <widget class="CWizardTreeWidget" name="treeWidgetColors">
         <property name="rootIsDecorated">
          <bool>false</bool>
         </property>
         <property name="uniformRowHeights">
          <bool>true</bool>
         </property>
         <property name="allColumnsShowFocus">
          <bool>true</bool>
         </property>
         <column>
          <property name="text">
           <string>Type</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Color</string>
          </property>
         </column>
         <item>
          <property name="text">
           <string>Background</string>
          </property>
          <property name="flags">
           <set>ItemIsSelectable|ItemIsEditable|ItemIsEnabled</set>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Default</string>
          </property>
          <property name="flags">
           <set>ItemIsSelectable|ItemIsEditable|ItemIsEnabled</set>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Event</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="flags">
           <set>ItemIsSelectable|ItemIsEditable|ItemIsEnabled</set>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Notice</string>
          </property>
          <property name="flags">
           <set>ItemIsSelectable|ItemIsEditable|ItemIsEnabled</set>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Action</string>
          </property>
          <property name="flags">
           <set>ItemIsSelectable|ItemIsEditable|ItemIsEnabled</set>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Inactive</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Alert</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Highlight</string>
          </property>
          <property name="flags">
           <set>ItemIsSelectable|ItemIsEditable|ItemIsEnabled</set>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Timestamp</string>
          </property>
          <property name="flags">
           <set>ItemIsSelectable|ItemIsEditable|ItemIsEnabled</set>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Link</string>
          </property>
          <property name="flags">
           <set>ItemIsSelectable|ItemIsEditable|ItemIsEnabled</set>
          </property>
         </item>
         <item>
          <property name="text">
           <string>White</string>
          </property>
          <property name="flags">
           <set>ItemIsSelectable|ItemIsEditable|ItemIsEnabled</set>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Black</string>
          </property>
          <property name="flags">
           <set>ItemIsSelectable|ItemIsEditable|ItemIsEnabled</set>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Blue</string>
          </property>
          <property name="flags">
           <set>ItemIsSelectable|ItemIsEditable|ItemIsEnabled</set>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Green</string>
          </property>
          <property name="flags">
           <set>ItemIsSelectable|ItemIsEditable|ItemIsEnabled</set>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Red</string>
          </property>
          <property name="flags">
           <set>ItemIsSelectable|ItemIsEditable|ItemIsEnabled</set>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Brown</string>
          </property>
          <property name="flags">
           <set>ItemIsSelectable|ItemIsEditable|ItemIsEnabled</set>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Purple</string>
          </property>
          <property name="flags">
           <set>ItemIsSelectable|ItemIsEditable|ItemIsEnabled</set>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Orange</string>
          </property>
          <property name="flags">
           <set>ItemIsSelectable|ItemIsEditable|ItemIsEnabled</set>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Yellow</string>
          </property>
          <property name="flags">
           <set>ItemIsSelectable|ItemIsEditable|ItemIsEnabled</set>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Light Green</string>
          </property>
          <property name="flags">
           <set>ItemIsSelectable|ItemIsEditable|ItemIsEnabled</set>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Cyan</string>
          </property>
          <property name="flags">
           <set>ItemIsSelectable|ItemIsEditable|ItemIsEnabled</set>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Light Cyan</string>
          </property>
          <property name="flags">
           <set>ItemIsSelectable|ItemIsEditable|ItemIsEnabled</set>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Light Blue</string>
          </property>
          <property name="flags">
           <set>ItemIsSelectable|ItemIsEditable|ItemIsEnabled</set>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Pink</string>
          </property>
          <property name="flags">
           <set>ItemIsSelectable|ItemIsEditable|ItemIsEnabled</set>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Gray</string>
          </property>
          <property name="flags">
           <set>ItemIsSelectable|ItemIsEditable|ItemIsEnabled</set>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Light Gray</string>
          </property>
          <property name="flags">
           <set>ItemIsSelectable|ItemIsEditable|ItemIsEnabled</set>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabAliases">
      <attribute name="title">
       <string>Aliases</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <property name="margin">
        <number>4</number>
       </property>
       <property name="spacing">
        <number>4</number>
       </property>
       <item row="0" column="0">
        <widget class="QToolButton" name="toolButtonAddAliases">
         <property name="text">
          <string>&amp;Add...</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>138</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>&amp;Filter:</string>
         </property>
         <property name="buddy">
          <cstring>lineEditAliasesFilter</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QLineEdit" name="lineEditAliasesFilter"/>
       </item>
       <item row="1" column="0" colspan="4">
        <widget class="QTreeView" name="treeViewAliases">
         <property name="editTriggers">
          <set>QAbstractItemView::AllEditTriggers</set>
         </property>
         <property name="rootIsDecorated">
          <bool>false</bool>
         </property>
         <property name="uniformRowHeights">
          <bool>true</bool>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
         <property name="allColumnsShowFocus">
          <bool>true</bool>
         </property>
         <attribute name="headerStretchLastSection">
          <bool>false</bool>
         </attribute>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonOK">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonApply">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CWizardTreeWidget</class>
   <extends>QTreeWidget</extends>
   <header>wizardtreewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../Resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButtonCancel</sender>
   <signal>clicked()</signal>
   <receiver>DialogIrcSettings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>271</x>
     <y>275</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
